stock_picking_backorder_strategy/__manifest__.py

```diff
@@ -1,36 +1,20 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Authors: Laetitia Gangloff
-#    Copyright (c) 2015 Acsone SA/NV (http://www.acsone.eu)
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    "name": "Picking backordering strategies",
-    "version": "8.0.0.1.0",
-    'author': "Acsone SA/NV, Odoo Community Association (OCA)",
-    "category": "Warehouse Management",
-    "website": "http://www.acsone.eu",
-    "depends": ["stock",
-                ],
-    "data": ["views/stock_views.xml",
-             ],
-    "demo": [],
-    "license": "AGPL-3",
-    'installable': False,
-    "application": False,
+    'name': 'Picking backordering strategies',
+    'version': '10.0.0.1.0',
```

Comment:
isn't `X.0.1.0.0` the default?

Reply:
@lreficent Yes of course. I'll change it